app/views/pets/show.html.erb

```diff
@@ -5,6 +5,9 @@
 <p>Approximate Age: <%= @pet.approximate_age %></p>
 <p>Sex: <%= @pet.sex %></p>
 <p>Adoption Status: <%= @pet.adoption_status %></p>
+<% if @pet.applicant %>
+  <p>Applicant: On hold for <%= @pet.applicant %></p>
```

Comment:
Does it make sense to have the page say 'applicant' or should it just say "On hold for ..."?

Reply:
The user story says "On hold for".